src/components/Wrapper.jsx

```diff
@@ -0,0 +1,18 @@
+import React from "react";
+import PropTypes from "prop-types";
+
+export const Context = React.createContext();
+
+// detect the language preference of the user's browser
+export const locale =
+  window.navigator.userLanguage || window.navigator.language;
+
+const Wrapper = ({ children }) => (
+  <Context.Provider value={{ locale }}>{children}</Context.Provider>
+);
+
+Wrapper.propTypes = {
+  children: PropTypes.element.isRequired,
+};
+
+export default Wrapper;
```

Comment:
♻️ I think you could move this to the context directory, maybe rename it to contexts, then set it up similarly to the AuthContextProvider. Wrapper is a little too vague, I think.

Reply:
I'm having conflicts when trying to rename the Wrapper component to Context because I have a variable in the file by the same name -

export const Context = React.createContext();

Can I just rename that variable?  Wasn't sure what the convention was for this.

Reply:
Addressed: be7e2037f34444e6f39195a5048a3ca152098286